openlibrary/core/edits.py

```diff
@@ -325,3 +325,8 @@ def create_comment(cls, username: str, message: str) -> dict[str, str]:
             "message": message,
             # XXX It may be easier to update these comments if they had IDs
         }
+
+
+@public
+def get_counts_mode(mode='all', **kwargs):
+    return CommunityEditsQueue.get_counts_by_mode(mode)
```

Comment:
We may want to add e.g. 1 or 5 minutes of caching so every request to openlibrary isn't making a new call to the database

```
from openlibrary.core import cache
from openlibrary.utils import dateutil

@public
def get_counts_mode(mode='open', **kwargs):
    return cache.memcache_memoize(
        CommunityEditsQueue.get_counts_by_mode,
        "librarian_queue_counts",
        timeout=dateutil.MINUTE_SECS)(mode, **kwargs)
```